Raven.Database/Config/Categories/ServerConfiguration.cs

```diff
@@ -8,6 +9,19 @@ namespace Raven.Database.Config.Categories
 {
     public class ServerConfiguration : ConfigurationCategory
     {
+        /// <summary>
+        /// This much time has to pass before 
+        /// </summary>
+        [DefaultValue(15)]
+        [TimeUnit(TimeUnit.Seconds)]
+        [ConfigurationEntry(Constants.ConcurrentResourceLoadTimeout)]
+        [ConfigurationEntry(Constants.ConcurrentResourceLoadTimeout + "InSec")]
+        public TimeSetting ConcurrentResourceLoadTimeout { get; set; }
+
+        [DefaultValue(8)]
+        [ConfigurationEntry(Constants.MaxConcurrentResourceLoads)]
+        public int MaxConcurrentResourceLoads { get; set; }
+
```

Comment:
@myarichuk Please consider my comments:

1) we are moving away from the constants usage when we declare settings.. if we really need to get the key of a setting we have `RavenConfiguration.GetKey(x => x.CategoryName.Setting)`, however the current configuration handling limits such usages
2) please leave just the "InSec" name.. other settings still have old names configured (might be useful for 4.0 documentation purposes to write about introduced changes), however they are going to be removed as well (in 4.0 won't be backward compatible anyway)
3) `TenantsConfiguration` seems to be a better matched class for those new settings:
https://github.com/ayende/ravendb/blob/v4.0/Raven.Database/Config/Categories/TenantConfiguration.cs